pkg/cmd/start/main.go

```diff
@@ -77,6 +77,9 @@ func NewStartCommand() *cobra.Command {
 	cmd.Flags().String("es-provision", "auto", "Whether to auto-provision an Elasticsearch cluster for suitable Jaeger instances. Possible values: 'yes', 'no', 'auto'. When set to 'auto' and the API name 'logging.openshift.io' is available, auto-provisioning is enabled.")
 	viper.BindPFlag("es-provision", cmd.Flags().Lookup("es-provision"))
 
+	cmd.Flags().String("kafka-provision", "auto", "Whether to auto-provision a Kafka cluster for suitable Jaeger instances. Possible values: 'yes', 'no', 'auto'. When set to 'auto' and the API name 'kafka.strimzi.io' is available, auto-provisioning is enabled.")
```

Comment:
The values don't seem consistent with "true"/"false" constants defined previously.

Reply:
Good catch!